source/common/http/conn_manager_impl.cc

```diff
@@ -1167,6 +1167,14 @@ void ConnectionManagerImpl::ActiveStream::decodeHeaders(RequestHeaderMapSharedPt
     return;
   }
 
+  // Rewrites the host of CONNECT-UDP requests.
```

Comment:
Do these need to be guarded with a flag?

Reply:
That would probably a safer choice. I would like to know other reviewers' thoughts on this as well. Thanks for your comment, Sean.

Reply:
Done.